default/scripting/monster_designs/ShipDesignManifest.focs.txt

```diff
@@ -0,0 +1,36 @@
+ShipDesignManifest
```

Comment:
Could you add a comment at the start of these files, indicating what they are for?